pipeline/03-evaluate.R

```diff
@@ -142,75 +139,69 @@ gen_agg_stats <- function(data, truth, estimate, bldg_sqft,
       prior_near_total_av = sum({{ rsn_col }} / 10, na.rm = TRUE),
       estimate_total_av = sum({{ estimate }} / 10, na.rm = TRUE),
 
-      # Assessment-specific statistics
-      across(
-        .fns = rs_fns_list, {{ estimate }}, {{ truth }},
-        .names = "{.fn}"
-      ),
+      # Assessment-specific ratio stats
+      rs_lst = rs_fns_list %>%
+        map(., \(f) exec(f, {{ estimate }}, {{ truth }})) %>%
+        list(),
```

Comment:
This has the exact same output as before except that it outputs to a list column that must then be unpacked using `unnest_wider()` below. This is way less fragile and readable compared to the now-deprecated behavior of `across()`.